DOCU-3261 Makelog changelog entry (#5740)
Kong/docs.konghq.com: app/konnect/updates.md

from a single, cloud-based control plane, and provides a catalog of all deployed
services. [Try it today!](https://cloud.konghq.com/quick-start)

## June 2023

> **Makelog Integration**
> : Makelog integration will allow {{site.konnect_saas}} feature teams to publish product updates in time with the feature releases and also manage the in-product release annoucements based on feature flags. Once integrated, customers will see the latest feature updates easily and quickly in UI.
> 

Reviewer: ```suggestion
**Release announcements in Konnect**
: With Makelog integration, you can now see the latest feature and update announcements in the {{site.konnect_saas}} UI. 
```
I don't think that "makelog integration" is the customer-facing feature here. Isn't the idea that hey, now you can view release announcements directly in Konnect? I don't know that anyone needs to know that feature flags are involved, or that it's "quick and easy".
